api/src/services/files.ts

```diff
@@ -124,6 +125,50 @@ export class FilesService extends ItemsService {
 		return primaryKey;
 	}
 
+	/**
+	 * Sync file from filesystem and update it on database
+	 * If the file doesn't exists on database it will create it
+	 */
+	async sync(relativePath: string): Promise<any> {
```

Comment:
Little worried about the performance downside of doing this one at a time 🤔 Wondering if we should maybe try doing this in batches of `n` to improve import performance